common/src/latestDependencies/java/com/revenuecat/purchases/common/purchaseExtensions.kt

```diff
@@ -0,0 +1,25 @@
+package com.revenuecat.purchases.common
```

Comment:
I wonder if it's be worth having the file name reflect the fact that this is billing client 4. I see that the path does say `latestDependencies`, but like, when you're looking up a file by name, you might miss the path

Reply:
Yes, good idea. It's easier to look for the files. The thing with the path is that gradle will compile anything in `main` and in `latestDependencies` if the `latestDependencies` flavor is selected. It will compile anything in `unityIAP` and `main` if the `unityIAP` is selected. Pretty magical huh!